src/Dynamo.All.Net6.sln

```diff
@@ -136,6 +136,7 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamoSandbox", "DynamoSandbox\DynamoSandbox.csproj", "{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AssemblyRenamerCLI", "Tools\AssemblyRenamerCLI\AssemblyRenamerCLI.csproj", "{F382C3F8-C55C-4350-800A-3D13A94F8E13}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "InstallUpdate", "Tools\InstallUpdate\InstallUpdate.csproj", "{0ED387BC-17B5-49B7-9C1D-BF58A4A5CC4D}"
```

Comment:
can we just get rid of this project?

Reply:
Not sure, I thought you created a task for that. DynamoCrypto, etc.

Reply:
Will try tracking down the issue to see if it's related.